### PR TITLE
fix(query): resolve prefixed names and anonymous blank nodes in AST bridge (#569)

### LINKED ISSUE #569: [W3C] Baseline execution, triage and failure classification for SPARQL 1.0 and 1.1 test suites
### Summary
Execute the newly integrated W3C SPARQL 1.0 (PR corese-stack/corese-w3c#21 / Issue corese-stack/corese-w3c#19) and SPARQL 1.1 (Issue corese-stack/corese-w3c#20) test suites, produce the initial baseline conformance report, and systematically classify all test failures.

#### 1. Baseline Run
- Run `./gradlew test validateEarlReport` across SPARQL 1.0 and SPARQL 1.1 dynamic test suites.
- Generate the initial baseline reports:
  - `build/reports/w3c/w3c-report.json` (Structured dashboard data)
  - `build/reports/w3c/earl-report.ttl` (Normative W3C EARL 1.0 + PROV-O graph)
  - `build/reports/w3c/w3c-report-summary.txt` (Terminal summary)

#### 2. SPARQL 1.0 Conformance Triage (PR corese-stack/corese-w3c#21 / Issue corese-stack/corese-w3c#19)
- Validate baseline SPARQL 1.0 conformance (syntax, basic graph patterns `?s ?p ?o`, `OPTIONAL`, `UNION`, `FILTER`, `ORDER BY`, `DISTINCT`, `LIMIT`/`OFFSET`).
- Isolate any regression or edge-case divergences on existing query constructs for immediate resolution.
- Target a 100% pass rate on all applicable SPARQL 1.0 test cases.

#### 3. SPARQL 1.1 Classification and Mapping (Issue corese-stack/corese-w3c#20)
Classify each failing SPARQL 1.1 test case according to explicit scientific categories:

- **Category A: True Bugs and Regressions on Baseline**
  - Failures occurring on features already supported by the engine (term equality, basic BGP matching, RDF 1.1 datatype checks).
  - *Action*: Isolate and resolve immediately.

- **Category B: Unimplemented SPARQL 1.1 Features (`SkipKind.DEFERRED` -> `earl:untested`)**
  - Failures caused by language features under active implementation in Cycle #52.
  - *Mapping to corese-core implementation issues*:
    - `VALUES`, `BIND`, inline data -> corese-stack/corese-core#562
    - Built-in scalar functions and casting operators -> corese-stack/corese-core#563
    - Subqueries, `EXISTS`, `NOT EXISTS`, `MINUS` -> corese-stack/corese-core#564
    - `GROUP BY`, Aggregates (`COUNT`, `SUM`, `AVG`, etc.), `HAVING` -> corese-stack/corese-core#565
    - Property Paths (`/`, `^`, `|`, `*`, `+`, `!`) -> corese-stack/corese-core#566
    - SPARQL 1.1 Update execution engine (`INSERT`, `DELETE`, `LOAD`, Graph Management) -> corese-stack/corese-core#567

- **Category C: Optional and Unsupported Features (`SkipKind.NOT_APPLICABLE` -> `earl:inapplicable`)**
  - Tests covering non-standard or out-of-scope behaviors (e.g., Generalized RDF).

#### 4. Documentation and Traceability
- Document every deferred test case in `docs/W3C_TEST_EXCLUSIONS.md` with its exact manifest fragment identifier, technical rationale, and linked tracking issue.
- Register all exclusions in the dynamic test executor via `KNOWN_EXCLUSIONS_BY_ID` .
- Publish the baseline conformance score to the web dashboard (`https://corese-stack.github.io/corese-w3c/`).

### Dependencies
- **Blocked by**:
  - [x] https://github.com/corese-stack/corese-w3c/issues/19
  - [x] https://github.com/corese-stack/corese-w3c/issues/20
  - [x] https://github.com/corese-stack/corese-w3c/issues/24
- **Blocks**:
  - https://github.com/corese-stack/corese-core/issues/568
  - https://github.com/corese-stack/corese-core/issues/562

### PR DESCRIPTION
**Pull request by remiceres**
Closes https://github.com/corese-stack/corese-core/issues/569

### CODE REVIEW

File: src/main/java/fr/inria/corese/core/next/query/impl/sparql/bridge/SparqlAstToExpression.java [hunk 1]
```diff
@@ -129,17 +144,151 @@ private static String unquoteLexical(String lexical) {
...
+        }
+        return sb.toString();
+    }
+
+    private static int appendEscaped(StringBuilder sb, String str, int nextIdx) {
+        char next = str.charAt(nextIdx);
+        if (next == 'u' || next == 'U') {
+            int hexLen = (next == 'u') ? 4 : 8;
+            if (nextIdx + 1 + hexLen <= str.length()) {
+                String hex = str.substring(nextIdx + 1, nextIdx + 1 + hexLen);
```
Comment (abdessamad-abdoun) [inline src/main/java/fr/inria/corese/core/next/query/impl/sparql/bridge/SparqlAstToExpression.java:284]: Integer.parseInt can throw a raw NumberFormatException if the query contains a malformed Unicode escape (\uXYZZ). Consider wrapping it to give a meaningful error:


try {
    int codePoint = Integer.parseInt(hex, 16);
    sb.appendCodePoint(codePoint);
    return nextIdx + 1 + hexLen;
} catch (NumberFormatException e) {
    throw new IllegalArgumentException(
        "Invalid Unicode escape '\\u" + hex + "' in prefixed name local part", e);
}

**Review by abdessamad-abdoun — APPROVED** [review]
